handbook/engineering/batch-changes/ce-onboarding.md

```diff
@@ -0,0 +1,62 @@
+# Batch Changes CE onboarding
+
+Welcome to Batch Changes! This page is primarily for new members of the CE team, but it could be helpful to other teams, too! After going through these steps, you should:
+
+- understand what Batch Changes is
+- be able to present it to customers from a value, usage, and functional perspective
+- be comfortable demoing Batch Changes to developers and answering questions
+
+## Prerequisites
+General [CE onboarding](../../ce/onboarding.md)
+
+## Part 1: Discover Batch Changes
+- Watch the [demo video](https://www.youtube.com/watch?v=eOmiyXIWTCw) and checkout the [landing page](https://about.sourcegraph.com/batch-changes/)
+- What is Batch Changes, and why does it matter to customers? Read about [product positioning and messaging](../../marketing/batch_changes_positioning.md)
+- How to demo Batch Changes? Watch the [Batch Changes deep dive](https://docs.google.com/presentation/d/1CN3KQf1Hfdb4RO6FgBgKuiHK4ERcOAHPgVnOcBu-MPU/edit#slide=id.ga366db8d9b_0_116)
+- How does Batch Changes work? Read the Batch Changes documentation section of the [Supporting Batch Changes guide](./supporting-batch-changes.md#batch-changes-documentation)
+
+
+## Part 2: Use Batch Changes
+
+- If you are blocked at any time during onboarding, ask for help in [#batch-changes](https://sourcegraph.slack.com/archives/CMMTWQQ49).
+- Feel free to document your onboarding experience and share it back with the Bacth Changes team so that we can make this even better!
+- In this tutorial, we will use [k8s.sgdev.org](https://k8s.sgdev.org) as our environment. You should have access by default using SSO.
```

Comment:
Do we want them using this? Normally we use Demo; I almost never access k8s. (Maybe there's a reason they need to use k8s, though.)

Reply:
My initial thinking was avoiding creating many tests on demo. WDYT?

Reply:
Ah, fair—I don't feel super strongly about this; if we have them do this, though, we should have them record the final demo on Demo and just include a note about that + needing to generate new creds for the CLI to point at the Demo instance. (I want people to be comfortable demoing on Demo specifically.)

Reply:
OK, I'll move this to demo, makes sense to train in the real setup